internal/utils.go

```
@@ -0,0 +1,63 @@
+		logger.Debugf("Creating connection %d", i+1)
+		conn, err := http_connection.NewInstrumentedHttpConnection(stageHarness, TCP_DEST, fmt.Sprintf("client-%d", i+1))
+		if err != nil {
+			logFriendlyError(logger, err)
+			return nil, err
+		}
+		connections[i] = conn
+	}
+	return connections, nil
+}
+
+func repeatSingleConnection(stageHarness *test_case_harness.TestCaseHarness, connectionCount int, logger *logger.Logger) ([]*http_connection.HttpConnection, error) {
```

golangci-lint (1.64.8)

func `repeatSingleConnection` is unused

(unused)